src/Console/ClearCache.php

```diff
@@ -12,7 +12,7 @@ class ClearCache extends Command
      *
      * @var string
      */
-    protected $signature = 'page-cache:clear {slug? : URL slug of page to delete}';
+    protected $signature = 'page-cache:clear {slug? : URL slug of page to delete} {--force-clear}';
```

Comment:
Nice idea.

Maybe change the option to `recursive`. The `force` nomenclature doesn't really convey what it does.

Reply:
I'm happy with that, I'll update the PR. I only called it force-clear because you are forcing it to use the clear function instead of the forget function, but recursive makes more sense from the users point of view

Reply:
Done